ios/engine/KMEI/KeymanEngineTests/VersionTests.swift

```diff
@@ -63,6 +63,29 @@ class VersionTests: XCTestCase {
     XCTAssertTrue(complex.majorMinor == Version("11.0")!, "Did not properly trim off excess version components")
   }
 
+  func testEquals() {
+    let simple = Version("12.0")!
+    let other = Version("12.0")!
+
+    XCTAssertTrue(simple == other)
+
+    let longer = Version("12.0.0.0")!
+
+    XCTAssertTrue(simple == longer)
+
+    let unequal_1 = Version("12.0.0.1")!
+
+    XCTAssertFalse(simple == unequal_1)
+    XCTAssertFalse(longer == unequal_1)
+
+    let diffMajor = Version("13.0")!
+
+    XCTAssertFalse(simple == diffMajor)
+
+    // Tests in "optional" mode.  Was trickier than it would appear!
+    XCTAssertEqual(Version("12.0"), Version("12.0"))
+  }
```

Comment:
Would a test `Version("0")` == `Version("0.0")`?

Reply:
Yes.  Added those in temporarily, just to be sure.

<img width="760" alt="Screen Shot 2020-07-10 at 2 07 12 PM" src="https://user-images.githubusercontent.com/25213402/87126601-abc6f280-c2b6-11ea-88bf-55f15c01d713.png">